cluster/saltbase/salt/kube-addons/kube-addons.sh

```diff
@@ -134,6 +134,15 @@ for k,v in yaml.load(sys.stdin).iteritems():
 ''' < "${kube_env_yaml}")
 fi
 
+# Wait for the default service account
+token_found=""
+while [ -z "${token_made}" ]; do
```

Comment:
`token_made` feels like a bug. If you're going to take one tonight, #10523 or an updated version of this. (Speaking as tomorrow's oncall.)